src/modules/Dropdown/Dropdown.js

```diff
@@ -184,21 +188,19 @@ export default class Dropdown extends Component {
 
   componentWillReceiveProps(nextProps) {
     super.componentWillReceiveProps(nextProps)
-    // console.debug('Dropdown.componentWillReceiveProps()')
-    // console.log('changed props:', _.transform(nextProps, (res, val, key) => {
-    //   if (val !== this.props[key]) res[key] = val
-    // }, {}))
+    debug.groupCollapsed('componentDidUpdate()')
+    debug('changed props:', objectDiff(nextProps, this.props))
+    debug.groupEnd()
 
     if (nextProps.value !== this.state.value) {
       this.setValue(nextProps.value)
     }
   }
 
   componentDidUpdate(prevProps, prevState) { // eslint-disable-line complexity
-    // console.debug('Dropdown.componentDidUpdate()')
-    // console.log('changed state:', _.transform(this.state, (res, val, key) => {
-    //   if (val !== prevState[key]) res[key] = val
-    // }, {}))
+    debug.groupCollapsed('componentDidUpdate()')
```

Comment:
Yep, we're on the same page. In either case, I don't think env flags are enough to enable / disable logging. I'd like to have a transform the strips the debugs. Since, we could / should also strip propTypes and the static_meta property.